bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/Win32DPIUtils.java

```diff
@@ -124,7 +124,7 @@ public static Point pixelToPointAsLocation(Drawable drawable, Point point, int z
 	}
 
 	public static Point pixelToPointAsSize(Point point, int zoom) {
-		return pixelToPoint(point, zoom, RoundingMode.DOWN);
```

Comment:
@HeikoKlare now as pixelToPointAsLocation and pixelToPointAsSize do the same thing, do we even need these separate methods?

Reply:
We do not necessarily need them, but we already embedded the distinction at all callers and we would of course still like to find a better concept for rounding the values, so to work on that it may be easier to keep the distinction for now. Otherwise, for the same reasons, you could also get rid of all the `RoundingMode` and just have a special handling for the case of `computeSize()`.

Reply:
sounds fair